seedlet/assets/sass/base/_reset.scss

```diff
@@ -110,10 +110,12 @@ a {
 }
 
 // Focus styles
-*:focus {
-	outline-width: 1px;
-	outline-style: dotted;
-	outline-color: var(--global--color-secondary);
+.site {
```

Comment:
Super-minor comment, but since this wrapper only contains one set of rules, I don't think it's necessary to nest these rules. Easier to just write `.site *:focus { ...`